src/Bswup/Bit.Bswup.Demo/Client/wwwroot/app.css

```
@@ -1,96 +1,147 @@
-    --code-text: #d7e1f0;
-    --inline-code-bg: #1b2940;
-    --inline-code-text: #ff8fb8;
-    --shadow: 0 1px 3px rgba(0, 0, 0, 0.4), 0 8px 24px rgba(0, 0, 0, 0.3);
+    --bit-shp-brd-radius: 0.125rem;
+    --bit-shp-brd-width: 0.0625rem;
+    --bit-mot-duration: 200ms;
+    --bit-mot-duration-short: 100ms;
+    --bit-mot-easing: cubic-bezier(0.33, 0, 0.67, 1);
+
+    /* The docs shell's own decisions, matching the BlazorUI demo's Styles/app.scss. */
+    --demo-font-sans: "Segoe UI", "Segoe UI Web (West European)", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)